inference-engine/thirdparty/clDNN/kernel_selector/core/kernel_base.cpp

```diff
@@ -8,7 +8,8 @@
 
 namespace kernel_selector {
 const primitive_db KernelBase::db;
-thread_local size_t KernelBase::counter = 0;
+// thread_local size_t KernelBase::counter = 0;
+std::atomic<size_t> KernelBase::counter(0);
```

Comment:
now that we are using processing order as unique id, we no longer need this counter isnt it?